metrics-sample-set/metrics-sample-set.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+module.exports = {
+  name: 'Decide if the user is in the metrics sample set',
+  startDate: '2015-01-01',
+  subjectAttributes: ['sampleRate', 'uniqueUserId'],
+  independentVariables: ['isInSampleSet'],
+  eligibilityFunction: function (subject) {
+    return true;
+  },
+  groupingFunction: function (subject) {
+    return {
+      isInSampleSet: !! this.bernoulliTrial(subject.sampleRate, subject.uniqueUserId)
```

Comment:
I think it makes more sense to define it in the experiment, migrating where the value is declared would allow us to rip out some code from FxA's config.

I see the error rate in metrics/errors.js is defined within the experiment instead of using this pattern. I'm happy to update.